### PR TITLE
Switch to use d2l-card instead of d2l-tile.

### PR DESCRIPTION
**Pull request by dbatiste**
* switches to use d2l-card instead of d2l-image-tile
* use built-in styles for consistency

### CODE REVIEW

File: src/d2l-dashboard-tile.html [hunk 1]
```diff
@@ -14,65 +17,39 @@
 <dom-module id='d2l-dashboard-tile'>
 	<template strip-whitespace>
 		<style>
-			.dashboard-text {
-				padding-top: 30px;
-				text-align: center;
-				flex: 1;
-				overflow: hidden;
-				word-wrap: break-word; /* IE/Edge */
```
Comment (dbatiste) [inline src/d2l-dashboard-tile.html]: I'm not 100% certain if these were needed.

File: src/d2l-dashboard-tile.html [hunk 2]
```diff
@@ -14,65 +17,39 @@
...
+				width: 100%;
+			}
+			d2l-card-content-title {
+				text-align: center;
 			}
-			d2l-image-tile {
-				--d2l-image-tile-image-height: 125px;
-				border-color: transparent;
+			d2l-card-content-meta {
+				margin-top: 12px;
```
Comment (dbatiste) [inline src/d2l-dashboard-tile.html]: This value is 2px larger than original, to adhere to base 6 layout guide.  I reached out to designers for consistent value for this but have not gotten a response yet.

File: src/d2l-dashboard-tile.html [hunk 3]
```diff
@@ -14,65 +17,39 @@
...
-				img-url="[[_imgUrl]]"
-				dropdown-label="Options"
-				hover-effect="emphasize-image lower-menu"
-			>
-				<d2l-dropdown-menu slot="d2l-image-tile-dropdown">
+		<d2l-card href="[[_link]]" text="[[_name]]">
+
+			<img slot="header" alt="" class="d2l-dashboard-tile-image" src="[[_imgUrl]]" />
+
+			<d2l-dropdown-more slot="actions" translucent visible-on-ancestor text="Options">
```
Comment (dbatiste) [inline src/d2l-dashboard-tile.html]: I noticed there is this hard-coded "Options" text.  I simply copied the value, but we should really localize this.

File: src/d2l-dashboard-tile.html [hunk 4]
```diff
@@ -14,65 +17,39 @@
...
 						<d2l-menu-item
 							id='change-display-name-option'
 							text="[[localize('changeDisplayName')]]"
 							on-d2l-menu-item-select="_changeDisplayName" />
 					</d2l-menu>
 				</d2l-dropdown-menu>
+			</d2l-dropdown-more>
+
+			<div slot="content">
+				<d2l-card-content-title>[[_name]]</d2l-card-content-title>
```
Comment (dbatiste) [inline src/d2l-dashboard-tile.html]: These components have baked in styles for title and meta data of cards.

**Comment by LiyueWang** [comment]
Thanks! @mtjandra

**Comment by dbatiste** [comment]
Thanks Miles.

Yeah, that's currently expected for cards (no underline or zoom effect).  The zoom effect was also removed from the My Courses card/tiles.

That being said, there are a couple of tweaks that I think will be made to the normal card.  Bob, Alex, and Jeff are currently deciding.  The changes may be: add the y-translation on hover , and possibly adding box-shadow (same as subtle card).  The focus effect applies the light blue border already, which is consistent with other components.  I'll defer merging this until those changes are decided and complete (next day or so hopefully).

**Comment by dbatiste** [comment]
I'll take a look in IE to see what's going wrong.

**Comment by dbatiste** [comment]
I think this should be ready.

**Comment by mtjandra** [comment]
Thanks @dbatiste . I do notice the images in IE/Edge look slightly stretched width wise. But I'm willing to accept it given it is IE/Edge.

**Pull request merged**